### PR TITLE
Add EIP: Soulbound Tokens

### PR DESCRIPTION
**Pull request by ogarciarevett**
## Simple Summary

Introducing the concept of "Soulbound Tokens" with explicit modifiers to restrict token transferability, enhancing security and control in blockchain applications or blockchain games.

## Abstract

This Ethereum Improvement Proposal (EIP) outlines a framework for implementing "Soulbound Tokens" to explicitly restrict the transfer of tokens, token batches, and addresses. This explicit approach enhances the security and transparency of blockchain applications, particularly in use cases such as NFTs, gaming assets, and governance tokens.

## Motivation

In the blockchain space, the ownership and transfer of tokens are fundamental concepts. However, there are scenarios where explicit control over transferability is required. For these situations, explicit restrictions offer a more secure, clear, and flexible approach compared to implicit solutions.

This EIP aims to standardize the implementation of Soulbound Tokens, offering a framework for token binding and explicit control. It provides developers with a versatile solution to secure token ownership.

### CODE REVIEW

File: EIPS/eip-7524.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+eip: 7524
```
Comment (abcoathup) [inline EIPS/eip-7524.md:2]:
```suggestion
eip: 7537
```
Assigning next sequential EIP/ERC number.  ERC7524 is already in use in #7775
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Please check existing EIPs. There are a number of standards that do exactly the same thing as this.

**Comment by ogarciarevett** [comment]
@Pandapip1 Please provide me one that have this, please

**Comment by abcoathup** [comment]
Internet search shows some of the existing Soulbound Token ERCs:
https://search.brave.com/search?q=site%3Aeips.ethereum.org%2FEIPS+soulbound

ERC5192: Minimal Soulbound NFTs
ERC5484: Consensual Soulbound Tokens
ERC5114: Soulbound Badge
ERC4973: Account-bound Tokens
ERC5633: Composable Soulbound NFT, EIP-1155 Extension
ERC5727: Semi-Fungible Soulbound Token
ERC4671: Non-Tradable Tokens Standard
ERC5516: Soulbound Multi-owner Tokens
ERC5851: On-Chain Verifiable Credentials
ERC6239: Semantic Soulbound Tokens

**Comment by ogarciarevett** [comment]
@abcoathup Thanks! The main difference is that any of those EIP's has an implementation, just the events.

**Comment by steven2308** [comment]
> @abcoathup Thanks! The main difference is that any of those EIP's has an implementation, just the events.

[ERC-6454](https://eips.ethereum.org/EIPS/eip-6454) actually has implementation and test cases. But is meant to be minimal, this one seems too complex with no need.

**Comment by abcoathup** [comment]
@ogarciarevett & @steven2308 suggest putting your comparison of other soulbound token ERCs in the discussion thread on Eth Magicians so people can see your points on why you think another soulbound ERC is needed..

**Comment by ogarciarevett** [comment]
@steven2308 that only works for erc721, this erc works for erc1155 and erc721 and even erc20 if you need it.

**Comment by SamWilsn** [comment]
I am closing this pull request because we are in the process of separating EIPs and ERCs into distinct repositories. Unfortunately, as far as we are aware, GitHub does not provide any tools to ease this migration, so every pull request will need to be re-opened manually.

As this is a PR to create / modify an ERC, I will kindly ask you to redirect this to the new repository at [ethereum/ERCs](https://github.com/ethereum/ERCs). **We have prepared [a guide](https://github.com/ethereum/ERCs/issues/8) to help with the process.**

If there is relevant history here, please link to this PR from the new pull request.

On behalf of the EIP Editors, I apologize for this inconvenience.

**Pull request closed without merging**